tests/integration-tests/conftest.py

```diff
@@ -171,6 +171,10 @@ def pytest_addoption(parser):
         "--slurm-database-stack-name",
         help="Name of CFN stack providing database stack to be used for testing Slurm accounting feature.",
     )
+    parser.addoption(
+        "--external-shared-storage-stack-name",
```

Comment:
Let's document this new option in the README at `tests/integration-tests/README.md` like we did for other options.

Reply:
Yes, I'll do this in my next commit